script.js

```diff
@@ -3,31 +3,54 @@ const charCount = document.getElementById("charCount");
 const submitBtn = document.getElementById("submitBtn");
 const maxChars = 250;
 
+
 tweetInput.addEventListener("input", () => {
   let inputLength = tweetInput.value.length;
 
+  if (inputLength === 0) {
+    submitBtn.classList.add("off");
+    submitBtn.disabled = true;
+  }
+   else 
+   {
+    submitBtn.classList.remove("off");
+    submitBtn.disabled = false;
+  }
+
   if (inputLength >= maxChars) {
     tweetInput.value = tweetInput.value.substring(0, maxChars);
-    inputLength = maxChars;
+    inputLength = maxChars; // تحديث العد بعد القطع
```

Comment:
I think we can move this to the top of the listener, this way we don't have to mutate `inputLength` (we try not to mutate variables)